削除機能実装

Mihara3389 opened the pull request:
＜気になる点＞
・DeleteControllerのロジックが長い点。
→何かまとめる方法や省略できるところがあればアドバイスいただきたいです。
・JPAでの@onetomanyがうまく実装できず、ロジックにて対応
→Daoを直接実装するしかないでしょうか。何か他方法があれば教えていただきたいです。

Mihara3389 committed:
削除機能実装

yoshizaki-takeshi commented on AutoSystem_springboot/src/main/java/com/example/service/AnswersService.java:
```diff
@@ -31,4 +31,12 @@
+   * 
+   */
+  public void deleteAnswer(Integer question_id) {
+	  answersRepository.deleteById(question_id);
```
ここの引数ってPKのidになるんじゃないですかね…？

Mihara3389 committed:
・テスト表示と採点機能をひとまとめに（TestController)
・Topの遷移をloginの遷移と同じに(AuthenticationController)
・採点履歴画面に(HistoryController)
・重複した設定かつ、エラーになるため設定を追加(application.properties)

Mihara3389 committed:
PRでの指摘事項反映：
correct_answersのpkがidではなく、question_idになっていたため修正

Mihara3389 committed:
Top画面の遷移機能を１つのControllerへまとめる

Mihara3389 committed:
・パラメータの書き方修正
・テスト採点結果はControllerを別にする

Mihara3389 committed:
Commonを新規作成し、同一処理を共通クラスにて実施
